apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -13,9 +13,184 @@
 // limitations under the License.
 
 using Google.Api.Gax.Grpc;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
 
 namespace Google.Cloud.Bigtable.V2
 {
+    public partial class BigtableClient
+    {
+        /// <summary>
+        /// Mutates a row atomically. Cells already present in the row are left
+        /// unchanged unless explicitly changed by <paramref name="mutations"/>.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Note that string is implicitly convertible to <see cref="BigtableByteString"/>, so <paramref name="rowKey"/> can
+        /// be specified using a string as well and its UTF-8 representations will be used.
+        /// </para>
+        /// <para>
+        /// This method simply delegates to <see cref="MutateRow(MutateRowRequest, CallSettings)"/>.
+        /// </para>
+        /// </remarks>
+        /// <param name="tableName">
```

Comment:
Specify which values can't be null, and validate them?

Reply:
(And for simplicity, maybe extract the MutateRowRequest construction to a method?)

Reply:
Done and done.